bin/deepstate/common.py

```diff
@@ -118,7 +118,7 @@ def parse_args(cls):
         help="Number of workers to spawn for testing and test generation.")
 
     parser.add_argument(
-        "--output_test_dir", type=str, required=False,
+        "--output_test_dir", default="out", type=str, required=False,
```

Comment:
I like this change, but if we could try to make logically independent, cross-cutting API changes like this in their own commit, it would be helpful when reviewing (and potentially debugging later).

Reply:
(I went ahead and separately committed just this change on its own, so don't worry about this case)